thingsboard_gateway/connectors/mqtt/json_mqtt_uplink_converter.py

```diff
@@ -13,10 +13,10 @@
 #     limitations under the License.
 
 from re import search
-from time import time
 
 from simplejson import dumps
 
+from thingsboard_gateway.gateway.constants import *
```

Comment:
It is better to use exactly what you will use (in your case `SEND_ON_CHANGE_PARAMETER`).
`from thingsboard_gateway.gateway.constants import SEND_ON_CHANGE_PARAMETER`

Reply:
ok